scripts/build_sputnikvm.sh

```diff
@@ -11,6 +11,24 @@ else
 	echo "With SputnikVM, running geth $OUTPUT ..."
 fi
 
+installrust() {
+    curl https://sh.rustup.rs -sSf | sh
+    source $HOME/.cargo/env
+}
+
+# Prompt to install rust and cargo if not already installed.
+if hash cargo 2>/dev/null; then
```

Comment:
Why `hash` and not something like `which`?

We could also check `$HOME/.cargo/env` existence, and use it if available.

Reply:
Portability. https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script

Rust users can configure their paths, ala https://stackoverflow.com/questions/38050995/how-can-the-location-of-cargos-configuration-directory-be-overridden, so checking a specific path isn't reliable. 

I think now we should use `command -v` instead of `hash`, though.